web/pdf_viewer.js

```diff
@@ -2345,9 +2357,7 @@ class PDFViewer {
     if (!this.pdfDocument) {
       return;
     }
-    if (mode === AnnotationEditorType.STAMP) {
-      this.#mlManager?.loadModel("altText");
-    }
+    this.#preloadEditingData(mode);
```

Comment:
The same line should *also* replace the following code: https://github.com/mozilla/pdf.js/blob/d6f63d0e4b35b31355bdf4f8338e284bf72c75b2/web/pdf_viewer.js#L936-L938